functions/AutoRemediateOrchestrator.js

```diff
@@ -2,17 +2,29 @@
 
 const CONFIG = require('./config')
 const AWS = require('aws-sdk')
+let delayedExecution;
+let delay;
+let queueURL;
+let type;
 
 module.exports.handler = (event, context, callback) => {
   console.log('Received event: ', JSON.stringify(event, null, 2))
-  console.log('Config settings: ', JSON.stringify(CONFIG, null, 2))
+  //console.log('Config settings: ', JSON.stringify(CONFIG, null, 2))
 
-  if (!event || !event.Records[0] || !event.Records[0].Sns || !event.Records[0].Sns.Message) {
+  if ((!event || !event.Records[0] || !event.Records[0].Sns || !event.Records[0].Sns.Message) && !event.Records[0].body) {
     callback(new Error('No event specified'))
     return
   }
-
-  let message = JSON.parse(event.Records[0].Sns.Message)
+  let message;
+  if(event.Records[0].body){
```

Comment:
This is confuse because line 14 will return an error if `event.Records[0].body` is invalid. If this line of code runs, the `else` condition is redudant.